src/resources/token/token.helper.js

```diff
@@ -0,0 +1,32 @@
+const { get } = require('lodash');
```

Comment:
Get rid of token.helper.

Helper should contain only simple (one-liner like) pieces of code.

Move all methods (getUserIdByToken, removeTokens, createTokens) to accessTokenService